conda/environment-arm64-flink.yml

```diff
@@ -0,0 +1,105 @@
+name: ibis-dev-flink
+channels:
+  - conda-forge
+dependencies:
+  # runtime dependencies
+  - python =3.10
+  - atpublic >=2.3
+  - bidict >=0.22.1
+  - clickhouse-connect >=0.5.23
+  - dask >=2022.9.1
+  - datafusion >=0.6
+  - db-dtypes >=0.3.0,<2
+  - deltalake
+  - duckdb-engine <1,>=0.1.8
+  - filelock >=3.7.0,<4
+  - fsspec >=2022.1.0
+  - gcsfs
+  - geoalchemy2 >=0.6.3
+  - geopandas >=0.6
+  - google-cloud-bigquery >=3,<4
+  - google-cloud-bigquery-storage >=2,<3
+  - impyla >=0.17
+  - multipledispatch >=0.6,<2
+  - numpy >=1.15,<2
+  - oracledb >=1.3.1
+  - packaging >=21.3
+  - pandas >=1.2.5
+  - parsy >=2
+  - pins >=0.8.2
+  - poetry-core >=1.0.0
+  - poetry-dynamic-versioning >=0.18.0
+  - polars >=0.19
+  - psycopg2 >=2.8.4
+  - pyarrow =11.0.0
+  - pyarrow-tests
+  - pyarrow-hotfix >=0.4
+  - pydata-google-auth
+  - pydruid >=0.6.5
+  - pymssql >=2.2.5
+  - pymysql >=1
+  - pyspark >=3
+  - python-dateutil >=2.8.2
+  - python-duckdb >=0.8.1
+  - python-graphviz >=0.16
+  - pytz >=2022.7
+  - regex >=2021.7.6
+  - requests >=2
+  - rich >=12.4.4
+  - shapely>=2,<3
+  - snowflake-connector-python >=3.0.2
+  - snowflake-sqlalchemy >=1.4.1
+  - sqlalchemy <3,>=1.4
+  - sqlalchemy-risingwave >=1.0.0
```

Comment:
It looks like a this environment file is the dev environment for before the `the-epic-split`. We should be able to remove all of the sqlalchemy-related dependencies here.

Reply:
Actually I will simply go through this file and add deletion suggestions